packages/pwa-kit-create-app/assets/bootstrap/js/package.json.hbs

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "{{answers.project.name}}",
+  "version": "0.0.1",
+  "license": "See license in LICENSE",
+  "engines": {
+    "node": "^16.0.0 || ^18.0.0",
+    "npm": "^7.0.0 || ^8.0.0 || ^9.0.0"
+  },
+  "ccExtensibility": {
+    "extends": "{{preset.templateSource.id}}",
+    "overridesDir": "overrides"
+  },
+  "devDependencies": {
+    "{{preset.templateSource.id}}": "{{answers.general.packageJSON.version}}"
+  },
+  "scripts": {
+    {{#each answers.general.packageJSON.scripts}}
+    "{{@key}}": "{{{script this}}}"{{#unless @last}},{{/unless}}
+    {{/each}}
+  },
+  "bundlesize": [
+    {
+      "path": "build/main.js",
+      "maxSize": "55 kB"
+    },
+    {
+      "path": "build/vendor.js",
+      "maxSize": "275 kB"
+    }
```

Comment:
@bendvc this needs to be changed, we're over this bundlesize now, sadly, so it'll fail a generated project right out of the gate